fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -2770,6 +2770,12 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static int lake_compaction_history_size = 20;
 
+    @ConfField(mutable = true)
+    public static long lake_min_compaction_interval_ms_on_success = 10000;
```

Comment:
chang to second?

Reply:
it will be more friendly to user

Reply:
I'm not sure whether the schedule interval can be milliseconds. If not, we can change it to second. What do you think?